app/src/main/java/ch/hikemate/app/model/authentication/FirebaseAuthRepository.kt

```diff
@@ -21,21 +21,6 @@ import kotlinx.coroutines.launch
 
 class FirebaseAuthRepository : AuthRepository {
 
-  /**
```

Comment:
Thanks for cleaning this up, the comments were indeed duplicated and unnecessary in the implementation of the interface. :)